back/engines/commercial/verification/spec/services/permissions/base_permissions_service_spec.rb

```diff
@@ -3,47 +3,47 @@
 require 'rails_helper'
```

Comment:
The permissions specs seemed an ideal candidate to use `let_it_be`, but this resulted in errors for me. I gave up, because I already spent a lot of time on this PR.

Reply:
The problem is that with `let_it_be`, you cannot have something (e.g. the service) use something else (e.g. the user) that is only defined later. So it doesn't seem possible to apply here.